Lab-4/patrik_svensson_lab_function_approximation.ipynb

```diff
@@ -0,0 +1,286 @@
+            "text/plain": [
+              "<Figure size 640x480 with 1 Axes>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          }
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "kl5VpJcfMgTf",
+        "colab_type": "text"
+      },
+      "source": [
+        "# **Results**\n",
+        "The results of the implemetation can be seen in the graph in the previous section."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_4GLBv0zWr7m",
+        "colab_type": "text"
+      },
+      "source": [
+        "# **Discussion**\n",
+        "The implementation of the algorithm is only quality assured by a visual comparison. It would be better if there was an quantitative method for assuring the correctness. Furthermore, I'm not sure if the implementation is right since the graphs diverge from each other."
```

Comment:
The implementation can be assessed through convergence and error, which is missing in your case.

Also, if the graphs diverge from each other, add a reflection as to why you think that is. As you have stated, you consider it a possibility that the implementation is incorrect and you should hence also revisit the theory to make sure your implementation is correct.